web/src/testgrid-grid-row-name.ts

```diff
@@ -29,6 +29,6 @@ export class TestgridGridRowName extends LitElement{
     @property() name: String;
 
     render(){
-        return html`${this.name}`;
+        return html`<span title="${this.name}">${this.name}</span>`;
```

Comment:
TIL!

That said, do we _want_ this to be a title? It changes the on-hover behavior and I don't think it helps for accessibility (from a brief search).

Reply:
Updated PTAL!